### PR TITLE
Add admin alert when a newer workarea version is available

### PR DESCRIPTION
**Pull request by mttdffy**
WORKAREA-107

### CODE REVIEW

File: admin/config/locales/en.yml [hunk 1]
```diff
@@ -2375,6 +2375,7 @@ en:
           product: product
           publishes_on_and_has_no_changes: "%{release_name} publishes on %{release_publish_date} and has no changes"
           variant: variant
+          workarea_version_outdated: A newer version of Workarea is availble! Contact your system integration team about upgrading.
```
Comment (bencrouse) [inline admin/config/locales/en.yml]:
```suggestion
          workarea_version_outdated: A newer version of Workarea is available! Contact your implementer about upgrading.
```

File: core/lib/workarea/latest_version.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+module Workarea
+  module LatestVersion
+    def self.get
+      Rails.cache.fetch('workarea/version/latest', expires_in: 3.days) do
+        ENV['HTTP_PROXY'] || ENV['HTTPS_PROXY']
```
Comment (bencrouse) [inline core/lib/workarea/latest_version.rb]: Is this just leftover cruft? `Net::HTTP` will pay attention to these, so I believe it isn't necessary to add them to the request.

Comment (mttdffy) [reply core/lib/workarea/latest_version.rb]: I based this on https://github.com/workarea-commerce/workarea/blob/master/core/lib/workarea/ping_home_base.rb#L9 and assumed there was some wizardry behind invoking the variables beforehand. If its cruft in both places I can remove

Comment (bencrouse) [reply core/lib/workarea/latest_version.rb]: It's gotta be cruft, go for it.

**Pull request merged**